internal/tiltfile/tiltfile.go

```diff
@@ -248,6 +259,29 @@ func stopBuild(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple,
 	return buildContext, nil
 }
 
+func callKustomize(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
+	var path string
```

Comment:
This should probably take a localPath instead of just a string.

Reply:
I dunno, what do y'all think? I'm not convinced.

Reply:
can you use the localPathFromSkylarkValue i recently added?

Reply:
Yeah I'll do that. I think we should do an audit of the Tiltfile built-ins that we currently have and which ones should take strings and which ones should take localPaths. Maybe only `read_file` should allow a string as a parameter?